utilities/bump_versions.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2018 Google LLC
```

Comment:
This version of python is this supported for? Please make sure that it runs well on 2.7 and 3.4+ normally.

Reply:
I just now tested on python 2.x and 3.x and it seems to work fine.